### DISCUSSION TITLE
折腾 - 1nan

### DISCUSSION THREAD

**Discussion by giscus** [yinan-c/pages-comment#6] [Announcements]
# 折腾 - 1nan

https://yinan.me/zheteng.html

**Comment by koobai** [comment]
折腾的乐趣就在于不停的折腾

**Reply by yinan-c** [reply]
哈哈，确实是这样，越折腾越起劲！

**Comment by ghost** [comment]
我的旧版本的博客是用的 Memos 方案，优点是：发布和编辑内容很方便。
现在的 timeline 用的是 Hugo 自带的 markdown 和 html 混用的方案，优点是可自定义的样式很多，缺点是发布和编辑内容麻烦。

**Reply by yinan-c** [reply]
感谢分享！期待发布更多有趣内容！

**Comment by huhexian** [comment]
不知道为什么，Pyboke，看到这个立马想到嫖客的拼音......

**Reply by yinan-c** [reply]
哈哈哈 🤣

**Comment by yinan-c** [comment]
刚刚根据这个帖子让 giscus 可以自动调节 light/dark，防止半夜进评论区被亮瞎眼睛！
https://github.com/giscus/giscus/issues/336

**Comment by iloveyz** [comment]
在用twikoo，感觉够用，好一段时间没有使用评论系统，发现twikoo有后端管理也挺方便就用了。自从用上了ChatGPT后，我甚至很多代码都交给了ChatGPT帮我写，包括我的博客...：）

**Reply by yinan-c** [reply]
ChatGPT 再帮我改 Pyboke 的代码的时候也帮了大忙
